backend/src/lib/services/http/index.js

```diff
@@ -0,0 +1,18 @@
+import { request } from 'gaxios'
+import logger from '../../../config/winston'
+
+const HttpService = {
+  post: ({ url, data }) => {
+    request({
+      url: url,
+      method: 'POST',
+      data
+    })
+      .then(logger.info(`Posted to ${url}`, { message: JSON.stringify(data) }))
+      .catch(err => {
+        logger.error(`Something went wrong with ${url}`, { message: err.message })
+      })
+  }
```

Comment:
I think we'll want to iterate on the success/error handling here. But because nothing else hinges on the outcome of this first post I've just left it as something that calls out and then logs a success or failure accordingly.

My thought with the HTTP wrapper is that if we decide one day `axios` or some other HTTP handler is the way to go, we only need change the package in one place and we can still have a consistent, tested approach to HTTP actions in the app.